vra-services-check/src/run_vrahealthtestsuite.py

```diff
@@ -0,0 +1,79 @@
+#import json
```

Comment:
Add a license header here.

Reply:
I do have '# Copyright 2018 VMware, Inc.  All rights reserved				
# The MIT license (the â€œLicenseâ€�) set forth below ...
at the top of the file -  is that not sufficient? thx

Reply:
Every source file needs the license and the copyright.